specs/pvs/corla.pvs

```diff
@@ -254,6 +271,9 @@ BEGIN
   full_hand_count_contest: TYPE FROM contest
 % @todo Contest name uniqueness.
 
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   outcome: TYPE
```

Comment:
I agree that we shouldn't call it an election outcome.  We are doing the audit contest-by-contest, and leaving some out, so we don't have an outcome for the election.

I'm presuming that the glossary is backing up this document, and that should probably be noted at the top. I guess we might also try to use convention to explicitly or implicitly refer to it in cases like this.

The glossary defines outcome in terms of a set of contest winners.  This actually gets trickier, since some elections have different classes of winner (e.g. Boulder city council 5 at-large seats, last-place winner gets a two-year term, others get 4-year terms). But I doubt that information will be discernable from the CVR file. For now I'd list that as a thorny issue that we may not get to this time around.

It looks like election_outcome, as a set of outcomes, might be this set of winners, in which case I'd suggest terms like `winner`, and `outcome`, to replace `outcome` and `election_outcome`. (Or, someday, an outcome could be a set of sets for each class_of_winners).